source/_drafts/Feature-Spotlight-Accessibility.md

```diff
@@ -0,0 +1,103 @@
+
+See the [resources section at the end of this post](#resources) for a list of screen readers.
+
+
+### How do you make a web application screen reader accessible?
+If you use the native elements for the purposes that they were intended you will be most of the way towards beings accessible. Using the native element is the recommendede way to make anything accessible for a screen reader. For instance if you use a `<button>` element you will get the following accessiblity attributes (without them actually being on the button):
+* `tabIndex` which allows users to tab to the button
+* `role="button"` which tells the screen reader that this is a button
+* The space and the enter key will both press the button
+
+In some cases, such as in Video.js, it will not be possible to use the native button element. So you will have to mimic the accessible functionality from the list above and use a `div`. For a `div` that wants to be classified as a button:
+* You have to add the `role="button"` attribute to classify it as a button.
+* You have to add a `tabIndex` which will allow the `div` to be navigated to using only the `tab` key
+* You have to add handling for the space and enter key that press the button
+
+{% pullquote %}
+A list of role attribute values can be found on [Mozilla Developer Network](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques#Composite_roles).
+{% endpullquote %}
+
+After adding roles to the controls and content in your webpage. The next thing to look over are `aria` attributes. For instance we use `aria-live` for our `ProgressBar` slider. This is because the `ProgressBar` is always updating when a video is playing.
+
+{% pullquote %}
+For a more complete list of [ARIA attributes see the specification](https://www.w3.org/TR/wai-aria-1.1/).
+{% endpullquote %}
+
+Finally you need to add an accessible "name" and "status" to an element so that it can be referred to correctly. For instance to indicate that the `MuteToggle` is not just a "button" we have to set the `title` and the `innerHTML`/`innerText` to "Mute Toggle". In Video.js we refer to accessible "name"/"status" updates as "ControlText". This will allow the screen reader to refer to the `MuteToggle` as "Mute Toggle" rather than button. We also have to update the button whenever it is "pressed" to indicate that it will now "Mute" or "Unmute" depending on its state.
+
+Here are some examples of accessiblity straight from Video.js:
+* The `MuteToggle` `<button>`:
+  * Has `aria-live` set to polite, which means that the screen reader should wait until the user is idle to send valuenow updates to the user
+  * Has "ControlText" of "Mute" or "Unmute" which indicates the current status of the button to the user. "ControlText" is a Video.js-ism that indicates a change in the `innerHTML` and `title` attribute that is localized along the way.
```

Comment:
The "ControlText" term was explained above. Not sure it needs to be explained again. Also, I think it only needs to be quoted once - when you're introducing a term we've invented and then explaining it.